marklogic-data-hub-central/ui-custom/src/components/TimelineVis/TimelineVis.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import "./TimelineVis.scss";
+import DateTime from "../DateTime/DateTime";
+import Value from "../Value/Value";
+import { getValByConfig } from "../../util/util";
+import _ from "lodash";
+import Timeline from "react-visjs-timeline";
+import "./TimelineVis.scss";
+import * as ReactDOMServer from 'react-dom/server';
+
+type Props = {
+    data?: any;
+    config?: any;
+};
+
+const COMPONENTS = {
+    DateTime: DateTime,
+    Value: Value,
+}
+
+/**
+ * Component for showing content on a timeline.
+ *
+ * @component
+ * @prop {object} config - Data table configuration object.
+ * @example
+ */
+const TimelineVis: React.FC<Props> = (props) => {
```

Comment:
Unless there is a reason to, let's just name this Timeline widget "Timeline" instead of "TimelineVis". To support this, can we import the Timeline widget from VisJS like this? 
```
import Timeline as TimelineVis from "react-visjs-timeline";
```

Reply:
Done!!